src/components/action/Action.tsx

```diff
@@ -17,6 +17,7 @@ export const MbAction = ({
   return (
     <button
       className={`action ${state} ${getFontType(size)} ${className}`}
+      disabled={!!(state === EState.DISABLED)}
```

Comment:
thank you so much for your contribute @marceloF5 

just a quick question, why are you forcing the !! when state has the same type?

Reply:
You're right, since the response of `===` is always a boolean. So `!!`  is unnecessary. I'll remove it.
Well spotted.

Reply:
Fixed it.